examples/nodejs/token-vending-machine/README.md

```diff
@@ -32,7 +32,7 @@ These three required configuration variables live in the [config.ts](./lambda/to
 First make sure to start Docker and install the dependencies in the `lambda` directory, which is where the AWS Lambda code lives.
 
 ```bash
-cd lambda
+cd lambda/token-vending-machine
```

Comment:
tvm

Reply:
oh wait this one is actually correct just wasn't expecting in this PR, but it's a good change so 👍